webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Button } from 'patternfly-react';
+import { syncInsights } from './InsightsCloudSyncActions';
+
+const InsightsCloudSync = () => (
+  <IntlProvider locale={navigator.language}>
```

Comment:
maybe we should to combine `webpack/InsightsCloudSync` and `webpack/ForemanInventoryUpload` into a one
`webpack/ForemanRhCloud` folder, so we won't have duplication of intl and other registries

Reply:
I suppose we can create sort of layout component for both pages. Right now it's only the `IntlProvider` component, so I suggest to leave this refactoring to a different PR